src/editors/containers/ProblemEditor/data/SettingsParser.js

```diff
@@ -0,0 +1,68 @@
+import { isEmpty, get, isNil } from 'lodash-es';
```

Comment:
```suggestion
import _ from 'lodash-es';
```

Here are some reasons why https://www.blazemeter.com/blog/import-lodash-libraries

Reply:
Changed